schema/codelists/awardFinalStatus.csv

```diff
@@ -0,0 +1,4 @@
+Code,Title,Description
+complete,Complete,"The award was made and the standstill period, if any, has ended."
+cancelled,Cancelled,"The award was cancelled by the buyer or the procuring entity (e.g. because of a change in needs, insufficient funds, or technical or procedural errors) after the bid submission deadline, but before the award was made or before the standstill period, if any, had ended."
```

Comment:
```suggestion
cancelled,Cancelled,"The award was cancelled by the buyer or the procuring entity (for example, because of a change in needs, insufficient funds, or technical or procedural errors) after the bid submission deadline, but before the award was made or before the standstill period, if any, had ended."
```